extensions/api.py

```diff
@@ -99,6 +99,13 @@ async def xkcd(self, ctx, args = None):
     async def lmgtfy(self, ctx, *query):
         await ctx.send(f"http://lmgtfy.com/?q={'+'.join(query)}")
 
+    @commands.command(name='lmddgtfy',
+                      description="give link for let me duck duck go that for you",
+                      brief="let me duck duck go that for you")
+    async def lmgtfy(self, ctx, *query):
```

Comment:
Same name as lmgtfy command
needs to change

Reply:
done colega :clap: :clap: